src/System.Private.CoreLib/src/System/Diagnostics/Eventing/FrameworkEventSource.cs

```diff
@@ -32,6 +32,7 @@ namespace System.Diagnostics.Tracing
     sealed internal class FrameworkEventSource : EventSource
     {
         // Defines the singleton instance for the Resources ETW provider
+        // This field wil be accessed through reflection by System.Runtime.WindowsRuntime
```

Comment:
Why does it need to access it through reflection? Using stuff via private reflection is not really decoupling it.